packages/localization/src/Deployer.js

```diff
@@ -121,6 +136,64 @@ const createTranslation = async (keyId, translationString) => {
   return _fetch(`/translations`, formData, HttpMethod.POST);
 };
 
+const deleteKey = async (keyId, keyName) => {
+  if (keyId != null && keyId !== '') {
+    _log(`Deleting key ${keyName} with id ${keyId}`);
+    return _fetch(`/keys/${keyId}`, null, HttpMethod.DELETE);
+  }
+};
+
+// finding unused keys inside our codebase
+
+const findFiles = async (key: string) => {
+  const app = await findInFiles.findSync(key, 'app');
+  const accessibility = await findInFiles.find(key, 'packages/accessibility');
+  const assets = await findInFiles.find(key, 'packages/assets');
+  const config = await findInFiles.find(key, 'packages/config');
+  const navigation = await findInFiles.find(key, 'packages/navigation');
+  const orbit = await findInFiles.find(key, 'packages/orbit');
+  const pdf = await findInFiles.find(key, 'packages/pdf');
+  const playground = await findInFiles.find(key, 'packages/playground');
+  const relay = await findInFiles.find(key, 'packages/relay');
+  const rnmodules = await findInFiles.find(key, 'packages/rnmodules');
+  const shared = await findInFiles.find(key, 'packages/shared');
+
+  return [
+    ...Object.keys(app),
+    ...Object.keys(accessibility),
+    ...Object.keys(assets),
+    ...Object.keys(config),
+    ...Object.keys(navigation),
+    ...Object.keys(orbit),
+    ...Object.keys(pdf),
+    ...Object.keys(playground),
+    ...Object.keys(relay),
+    ...Object.keys(rnmodules),
+    ...Object.keys(shared),
+  ];
+};
+
+const findUnusedKeys = async () => {
+  const unusedKeys = await Promise.all(
+    Object.keys(translations).map(async tr_key => {
+      const filesFoundWithQuotes = await findFiles(`'${tr_key}'`);
+      const filesFoundWithDoubleQuotes = await findFiles(`"${tr_key}"`);
+
```

Comment:
I think `findFiles(tr_key);` should be sufficient

it does not matter if it is in wrapped in single or double quotes

Reply:
Well, if you had:
`hello.world` - unused key
`hello.world.mate` - used key
then `hello.world` would not be detected as unused key.

Reply:
Ahh, of course